addon/create-class-computed.js

```diff
@@ -0,0 +1,93 @@
+import Ember from 'ember';
+import EmberObject from 'ember-object';
+import computed from 'ember-computed';
+import on from 'ember-evented/on';
+import observer from 'ember-metal/observer';
+import { isNone } from 'ember-utils';
+// import getOwner from 'ember-owner/get';
+import get from 'ember-metal/get';
+import { setProperties } from 'ember-metal/set';
+import WeakMap from 'ember-weakmap';
+import { mapKeysToValues } from './-build-computed';
+import getValue from './get-value';
+import collapseKeys from './collapse-keys';
+import flattenKeys from './flatten-keys';
+
+const { defineProperty } = Ember;
+
+const PROPERTIES = new WeakMap();
+
+function findOrCreatePropertyInstance(context, propertyClass, key) {
+  let propertiesForContext = PROPERTIES.get(context);
+  if (isNone(propertiesForContext)) {
+    propertiesForContext = {};
+    PROPERTIES.set(context, propertiesForContext);
+  }
+
+  let property = propertiesForContext[key];
+  if (property) {
+    return property;
+  }
+
+  // let owner = getOwner(context);
+  property = propertyClass.create(/*owner.ownerInjection(), */{
```

Comment:
If you don't have an owner in the property class you won't be able to e.g. inject service which is something that might be helpful at times.